android/KMEA/app/src/main/java/com/keyman/engine/KMKeyboard.java

```diff
@@ -232,11 +232,15 @@ public void initKMKeyboard(final Context context) {
 
     getSettings().setUseWideViewPort(true);
     getSettings().setLoadWithOverviewMode(true);
-    if (0 != (context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE)) {
+
+     if (
+       0 != (context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE)
+       || KMManager.getTier(null) != KMManager.Tier.STABLE
```

Comment:
https://github.com/keymanapp/keyman/blob/93dc41e2f5e1892f0de3fa821f44da8fbd144355/android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java#L339-L348

When left `null`, `.getTier` uses the current engine version.